app/controllers/flood_risk_engine/errors_controller.rb

```diff
@@ -0,0 +1,52 @@
+require_dependency "flood_risk_engine/application_controller"
+
+module FloodRiskEngine
+  class ErrorsController < ApplicationController
+
+    def show
+      render(
+        template: file_for(template),
+        locals: { message: exception.try(:message), enrollment: enrollment }
+      )
+    end
+
+    protected
+
+    def error
+      @error ||= params[:id]
+    end
```

Comment:
Is @error the error code eg 404? If so error_code might be a clearer name as error could be the exception object or a message